### DISCUSSION TITLE
GUI spamming me with errors when i open Pool tab

### DISCUSSION THREAD

**Discussion by Borgiz** [Chia-Network/chia-blockchain#7274] [Support]
When i open pool tab i receive this: 
'dict' object has no attribute 'get_current_derivation_record_for_wallet'
'dict' object has no attribute 'get_confirmed_balance_for_wallet'

No matter if i close them i keep receiving this errors every second. Don't know what to do with this. 
Running Chia on Windows 10 GUI 1.2.0

**Comment by loppefaaret** [comment]
try and reset your config as instructed below:
WINDOWS COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.2.0\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```

**Reply by Borgiz** [reply]
tried it today but it didnt help

**Reply by loppefaaret** [reply]
what kind of CPU are you using ?

**Reply by Borgiz** [reply]
intel i3-8100

**Comment by loppefaaret** [comment]
then there is only left to try and remove the databases, as that is the only thing constant
first try the wallet, if that dont help, try the blockchain:
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

WINDOWS BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Reply by Borgiz** [reply]
wow looks like it works! GUI stopped bombing me with this pop-ups. Now waiting for my wallet to sync and then i tell you if everything is ok. 
Thanks a lot for your time!

**Reply by Borgiz** [reply]
Finally got my wallet synced. Everything is working and no pop-ups, thanks again!
